graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/ReferenceSubqueryTest.java

```diff
@@ -109,8 +109,50 @@ void list() {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                var _a_customer_2168032777_address = _a_customer.address().as("address_2138977089");
+                                return DSL.row(
+                                        DSL.row(
+                                                DSL.multiset(
+                                                        DSL.select(queryForQuery_customer_d1_address())
+                                                        .from(_a_customer_2168032777_address)
+                                                        .orderBy(_a_customer_2168032777_address.fields(_a_customer_2168032777_address.getPrimaryKey().getFieldsArray()))
+                                                )
+                                        ).mapping(_iv_e -> _iv_e.map(Record1::value1))
+                                ).mapping(Functions.nullOnAllNull(Customer::new));
+                            }
+                            private static SelectField<Address> queryForQuery_customer_d1_address() {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                var _a_customer_2168032777_address = _a_customer.address().as("address_2138977089");
+                                var _a_address_2138977089_store = _a_customer_2168032777_address.store().as("store_1314257795");
+                                return DSL.row(
+                                        _a_customer_2168032777_address.getId(),
+                                        DSL.row(
+                                                DSL.multiset(
+                                                        DSL.select(queryForQuery_customer_d1_address_d2_stores())
+                                                        .from(_a_address_2138977089_store)
+                                                        .orderBy(_a_address_2138977089_store.fields(_a_address_2138977089_store.getPrimaryKey().getFieldsArray()))
+                                                )
+                                        ).mapping(_iv_e -> _iv_e.map(Record1::value1))
+                                ).mapping(Functions.nullOnAllNull(Address::new));
+                            }
+                            private static SelectField<Store> queryForQuery_customer_d1_address_d2_stores() {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                var _a_customer_2168032777_address = _a_customer.address().as("address_2138977089");
+                                var _a_address_2138977089_store = _a_customer_2168032777_address.store().as("store_1314257795");
+                                return DSL.row(_a_address_2138977089_store.getId()).mapping(Functions.nullOnAllNull(Store::new));
+                            }
+                        """
```

Comment:
```suggestion
                ".select(queryForQuery_customer_d1_address()).from(_a_customer_2168032777_address)",
                ".select(queryForQuery_customer_d1_address_d2_stores()).from(_a_address_2138977089_store)"
```
Ser ut til at testene egentlig bare brydde seg om aliasene ble funnet/brukt i det hele tatt, men vi kan legge på å sjekke at de er på rett plass i koden som i min suggestion over.